blog/urls.py

```diff
@@ -0,0 +1,30 @@
+from django.urls import path
+
+from .views import (
+    PostListView,
+    PostDetailView,
+    CommentaryCreateView,
+)
+
+urlpatterns = [
+    path(
+        "",
+        PostListView.as_view(),
+        name="index",
+    ),
+
+    path(
+        "posts/<int:pk>/",
+        PostDetailView.as_view(),
+        name="post-detail",
+    ),
+
+    path(
+        "posts/<int:pk>/create",
```

Comment:
Initially I thought that the purpose of this endpoint is posts creation, not commentaries. Rename endpoint to a more descriptive name and add slash to the end